src/main/java/mate/academy/spring/controller/MovieSessionController.java

```diff
@@ -47,10 +47,9 @@ public MovieSessionResponseDto addMovieSession(@RequestBody MovieSessionRequestD
     }
 
     @GetMapping("/available")
-    public List<MovieSessionResponseDto> getAllAvailableSessions(@RequestParam Long id,
-                                                          @RequestParam
-                                                          @DateTimeFormat(pattern = "dd.MM.yyyy")
-                                                            LocalDate date) {
+    public List<MovieSessionResponseDto>
+            getAllAvailableSessions(@RequestParam Long id, @RequestParam
```

Comment:
```suggestion
            getAllAvailableSessions(@RequestParam Long movieId, @RequestParam
```

Reply:
this is the name of the project, I did not change it